From f2555fc4401dab30cc36001457bb2529e4fe2107 Mon Sep 17 00:00:00 2001
Message-ID: <f2555fc4401dab30cc36001457bb2529e4fe2107.1732837202.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732837202.git.sharlatanus@gmail.com>
References: <cover.1732837202.git.sharlatanus@gmail.com>
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Sun, 24 Nov 2024 13:27:31 +0300
Subject: [PATCH 12/15] gnu: go-github-com-itchyny-gojq: Install the library.

* gnu/packages/web.scm (go-github-com-itchyny-gojq): Install the library
files instead of "gojq" binary.
[arguments]: Remove <unpack-path>, adjust <import-path>.

Change-Id: I72235e0cfeee81b0e7490ebd9d92d0d36ceb1844
Reviewed-by: Sharlatan Hellseher <sharlatanus@gmail.com>
---
 gnu/packages/web.scm | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gnu/packages/web.scm b/gnu/packages/web.scm
index d7c25e8b5b..7e355b8d77 100644
--- a/gnu/packages/web.scm
+++ b/gnu/packages/web.scm
@@ -5605,8 +5605,7 @@ (define-public go-github-com-itchyny-gojq
     (build-system go-build-system)
     (arguments
      (list
-      #:import-path "github.com/itchyny/gojq/cmd/gojq"
-      #:unpack-path "github.com/itchyny/gojq"))
+      #:import-path "github.com/itchyny/gojq"))
     (inputs
      (list go-github-com-google-go-cmp
            go-github-com-itchyny-timefmt-go
-- 
2.46.0

